tests/scripts/check-python-files.sh

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env sh
+
+# This file is part of mbed TLS (https://tls.mbed.org)
+#
+# Copyright (c) 2018, ARM Limited, All Rights Reserved
+#
+# Purpose
+#
+# Run 'pylint' on Python files for programming errors and helps enforcing
+# PEP8 coding standards.
+
+if `hash pylint > /dev/null 2>&1`; then
+    pylint -j 2 tests/scripts/generate_test_code.py --rcfile .pylint
+    pylint -j 2 tests/scripts/test_generate_test_code.py --rcfile .pylint
+    pylint -j 2 tests/scripts/mbedtls_test.py --rcfile .pylint
+else
+    echo "$0: WARNING: 'pylint' not found! Skipping checks on Python files."
```

Comment:
I think it would be good if the script here returned a failure exit code to the system. I think that in the current way the script is written only the warning is printed by 0 is returned if pylint does not exist.

Reply:
It is deliberately done since pylint will be a new dependency and not everybody would like to add it to their dev env straight away. We can make it mandatory as we add more python file and get more strict. To be honest I was lazy to add new "--run-pylint" and "--skip-pylint" options in all.sh.